doozerlib/runtime.py

```
@@ -593,6 +593,15 @@ def filter_disabled(n, d):
                 self.image_name_map[name] = img.key
                 self.image_name_map[short_name] = img.key
 
+                name_in_bundle = img.data.get('name_in_bundle', None)
+                if name_in_bundle and name_in_bundle!=name and name_in_bundle!=name:
+                    if name_in_bundle in self.image_name_map:
+                        self.logger.warning(
```

I think we should raise an error if this happens.